### PR TITLE
README: update make command to build full release

### PR DESCRIPTION
**Pull request by coolaj86**
`make generate all` => `TAGS=bindata make generate all`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ This project has been
 
 From the root of the source tree, run:
 
-    make generate all
+    TAGS=bindata make generate all
```
Comment (techknowlogick) [inline README.md]: my opinion is that this change should remain as is because bindata is only optional, and shouldn't be enforced.

Comment (lafriks) [reply README.md]: @techknowlogick I would prefer to have it as all releases we provide are with it and it makes so much more problems if it is built without bindata

Comment (techknowlogick) [reply README.md]: ok. sounds good. LGTM.

**Review by techknowlogick — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lafriks) [inline README.md]: It should be `TAGS="bindata"` (with quotes) to keep consistency with other docs

**Review by lafriks — CHANGES_REQUESTED** [review]

**Review by jonasfranz — APPROVED** [review]

**Comment by lafriks** [comment]
It's not that I would mind much that this was merged but please wait till all change requests are approved before merging :)

**Pull request merged**